recipes/bytewax-feedstock/bld.bat

```diff
@@ -0,0 +1,7 @@
+@echo on
+SET RUST_BACKTRACE=1
+SET OPENSSL_DIR=%PREFIX%
+SET OPENSSL_LIB_DIR=%PREFIX%\lib
+SET OPENSSL_INCLUDE_DIR=%PREFIX%\include
+%PYTHON% -m pip install . -vv
+
```

Comment:
This file is not used b/c you are skipping Windows. Do you want to keep it and try again in the feedstock or do you want to remove it?

Reply:
The windows build fails, but I cannot figure out the reason from the logs: I was trying to see if I can get hold of a win machine in order to debug the build.

Reply:
You can skip it if the solution is too complex and open an issue in the feedstock to fix it later.